.github/pull_request_template.md

```diff
@@ -1,14 +1,25 @@
-## Context & Description
+<!--
+👋 Thank you for your Pull Request 🙏
+-->
+
+### Checklist
+
+- [ ] I have read [Contributing > Pull requests](https://tuono.dev/documentation/contributing/pull-requests)
+
```

Comment:
I don't like the checklist. Sounds brutal to me
```suggestion
👋 Thank you for your Pull Request 🙏
 Guidelines: [Contributing > Pull requests](https://tuono.dev/documentation/contributing/pull-requests)
-->
```

Reply:
It already happened that people opened PR with wrong title or delete PR template completely.
For this reason I think that we should add a checkbox so the contributing guideline are more visibile.
This will avoid to request a title update or updating the title ourselves.
It might redundant when opening multiple PR's but in the end it just adding a `x` in the template